test/workflow/scripts/app_cli.sh

```
@@ -0,0 +1,104 @@
+echo "[App Delete Start] ------------------------------------------------"
+osmo app delete ${APP_NAME} -a -f
+if [ $? -ne 0 ]; then
+    echo "[App Delete Failed] Failed to delete app"
+    exit 1
+fi
+echo "[App Delete Done]"
+
+sleep 60
+
+echo "[App Info Start] ------------------------------------------------"
+osmo app info ${APP_NAME}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)